xml/System.Text/StringBuilder+ChunkEnumerator.xml

```diff
@@ -14,11 +14,16 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>ChunkEnumerator supports both the IEnumerable and IEnumerator pattern so foreach 
-            works (see GetChunks).  It needs to be public (so the compiler can use it 
-            when building a foreach statement) but users typically don't use it explicitly.
-            (which is why it is a nested type).</summary>
-    <remarks>To be added.</remarks>
+    <summary>Supports simple iteration over the chunks of a <see cref="T:System.Text.StringBuilder" /> instance.</summary>
+    <remarks>  
+      <format type="text/markdown"><![CDATA[  
+
+A `ChunkEnumerator` is returned by the <xref:System.Text.StringBuilder.GetChunks%2A?displayProperty=nameWithType> method. It supports both the <xref:System.Collections.IEnumerable> and <xref:System.Collections.IEnumerator> patterns so that the chunks can be enumerated with `foreach` in C# or `For Each` in Visual Basic.
```

Comment:
```suggestion
## Remarks

A `ChunkEnumerator` is returned by the <xref:System.Text.StringBuilder.GetChunks%2A?displayProperty=nameWithType> method. It supports both the <xref:System.Collections.IEnumerable> and <xref:System.Collections.IEnumerator> patterns so that the chunks can be enumerated with `foreach` in C# or `For Each` in Visual Basic.
```